src/app_demo/should_be_generated/app/RequestReceiver.h

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 #define SRC_APP_DEMO_SHOULD_BE_GENERATED_APP_REQUESTRECEIVER_H_
 
 #include <INIReader.h>
+#include <grpcpp/ext/proto_server_reflection_plugin.h>
```

Comment:
why not move to cpp to reduce compile time.

Reply:
done